introduction.md

```diff
@@ -0,0 +1 @@
+Hello, new students of the dojo. Please learn the dojo rules, and lets start the training.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.